packages/wdio-applitools-service/applitools-service.d.ts

```diff
@@ -3,6 +3,7 @@
 declare module WebdriverIO {
     interface ServiceOption extends ApplitoolsConfig {}
     interface Browser extends ApplitoolsBrowser {}
+    interface Element {}
```

Comment:
why do you need this?

Reply:
Because here we define a new `WebdriverIO` namespace, that has new `Browser` and `ServiceOption`, and no `Element` defined. Then `WebdriverIO.Element` throws an error. I can try to find a better way.